maistra/scripts/update-deps.sh

```diff
@@ -94,15 +102,34 @@ function copy_files() {
 }
 
 function run_bazel() {
-  for config in s390x ppc x86_64; do
-    bazel --output_base="${OUTPUT_BASE}" build --nobuild --config="${config}" //src/... //test/... //extensions/...
+  # Fetch stats_plugin just to load emsdk and net_zlib dependencies
+  bazel --output_base="${OUTPUT_BASE}" fetch //extensions/stats:stats_plugin || true
+
+  # Patch emsdk and net_zlib
+  echo "*************  patch emsdk ****************"
+  pushd "${OUTPUT_BASE}/external/emsdk" 
+  patch -p1 -i ${PATCH_DIR}/emsdk.patch
+  popd
+
+  echo "*************  patch net_zlib ****************"
+  pushd "${OUTPUT_BASE}/external/net_zlib"
+  patch -p1  -i ${PATCH_DIR}/net_zlib.patch
+  popd
+
+  # Fetch all the rest and check everything using "build --nobuild "option
+  for config in s390x ppc x86_64; do   
+    bazel --output_base="${OUTPUT_BASE}" build --nobuild --config="${config}" //src/... //test/...  //extensions/...
   done
 }
 
 function main() {
+   echo "*************  validate ****************"
```

Comment:
nit: I'd move these echoes to under the functions, if they are still useful? (If not, just remove them)

Reply:
I'll remove them